jps-plugin/src/com/goide/jps/model/JpsGoSdkType.java

```diff
@@ -43,4 +43,9 @@ public JpsDummyElement createDefaultProperties() {
   private static File getExecutable(@NotNull String path, @NotNull String command) {
     return new File(path, getBinaryFileNameForPath(command));
   }
+
+  @NotNull
+  public static File getExecutableFileFromEnv(String command) {
```

Comment:
It's not a good idea to annotate with not null the method that can return a null value. Please keep this in mind next time.